CameraXVideo/app/src/main/res/layout-land/fragment_camera.xml

```diff
@@ -14,80 +14,115 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/container"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
     <androidx.camera.view.PreviewView
         android:id="@+id/previewView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"/>
-
+        android:background="@color/bkPurple"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintDimensionRatio="H,16:9"/>
```

Comment:
I would suggest removing all the code related to dimension ratio, here and in the Java code. Instead, rely on PreviewView's scaleType. e.g. `app:scaleType="fitCenter"` It's simpler that way.

Using dimension ratio is kinda duplicating PreviewView's fitting feature. Since this a code sample, we want to provide developers with the best practice.

Reply:
as we chatted off-line,  the UI part (Audio checkbox) might be more consistent with Aspect ratio approach.  Probably keep it for this case, and for other cases that having no UI controls above video, we should use the "fitCenter" to "fit video content into the preview window". It is a great point and we shall document the 2 approaches in the future documentation.